docusaurus/docs/cloud/projects/collaboration.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Collaboration
+displayed_sidebar: cloudSidebar
+description: Share your projects on Strapi Cloud to collaborate with others.
+canonicalUrl: https://docs.strapi.io/cloud/projects/collaboration.md
+sidebar_position: 1
+---
+
+# Collaboration on projects
+
+Projects are created by a user via his Strapi Cloud account, which is linked to their GitHub account. Strapi Cloud users can share their projects to anyone else, so these new users can have access to the Cloud dashboard and collaborate on that project, without the project owner to ever have to share their GitHub credentials.
+
+Users invited to collaborate on a project, called maintainers, do not have the same permissions as the project owner. Contrary to the latter, maintainers:
```

Comment:
```suggestion
Users invited to collaborate on a project, called maintainers, do not have the same permissions as the project owner. Contrary to the project owner, maintainers:
```

Suggesting it because, maybe it was because I reviewed it at the end of a working day 😂, I had to re-read to understand who "the latter" was referring to 😅